exercises/edward_brown/ex01_basics/src/ToLowerCase.cpp

```diff
@@ -0,0 +1,10 @@
+#include "ToLowerCase.h"
+#include <algorithm>
+#include <cctype>
+
+std::string toLowerCase(std::string MixedOrUpperCase) {
+  std::transform(MixedOrUpperCase.cbegin(), MixedOrUpperCase.cend(),
+                 MixedOrUpperCase.begin(),
+                 [](char C) -> char { return std::tolower(C); });
```

Comment:
Looks like that change got caught up in f4a8b5b8e33a8c2193f78109ff5b7785e29d4b5d.